src/Loaders/CollectionDataLoader.php

```diff
@@ -60,7 +60,7 @@ private function buildCollection($collection)
 
         return collect($this->filesystem->files($path))
             ->reject(function ($file) {
-                return Str::startsWith($file->getFilename(), '_');
+                return Str::startsWith($file->getFilename(), '_') || Str::startsWith($file->getFilename(), '.git');
```

Comment:
```suggestion
                return Str::startsWith($file->getFilename(), ['_', '.git']);
```

Reply:
TIL. I didn't know you could pass an array into the starts with function. That's much cleaner!